components/alert/alert.css

```diff
@@ -0,0 +1,50 @@
+.alert{
+    position:relative;
+    max-width:600px;
+    width: calc(100% - 0.5rem);
+    margin: 0.5rem;
+    padding: 1rem;
+    border-radius: 0.5rem;
```

Comment:
It would be good if you use px instead of rem for the value which is less than 10px

Reply:
I personally prefer using rem as it helps in responsiveness, and use px for very small values (generally where value < 8px)